canonical-websites/blog.ubuntu.com
# Add missing dependenncies

## Done

Add prometheus metrics to blog

## QA

- Check out this feature branch
- add `TALISKER_NETWORKS=172.17.0.1` to `.env`
- access a page http://0.0.0.0:8023/
- http://0.0.0.0:8023/_status/metrics
- make sure you have some gunicorn metrics
